build.gradle.kts

```diff
@@ -48,6 +51,6 @@ project.tasks.withType(KotlinJvmCompile::class.java) {
 }
 
 project.tasks.withType(Test::class.java) {
-    useJUnitPlatform()
+    useTestNG()
```

Comment:
Nice catch!

Reply:
haha, thanks!  I accidentally started coding it up for JUnit before I remembered the annoying licensing issues